notebook/mlops.ipynb

```diff
@@ -4,28 +4,57 @@
    "cell_type": "markdown",
```

Comment:
<p>This is nice. What about including links so the api code similar to <code>run.py</code> for users to look at whilst they are waiting?</p><p> </p><p>Also, perhaps its worth mentioning they can re-run to see the latest events, and once they see Endpoint CREATE_IN_PROGESS to move to the next step?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-sagemaker-safe-deployment-pipeline/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>